docs/content/guides/6.multistore/2.tooling-and-concepts/4.deployment/8.ci-cd.md

```diff
@@ -7,14 +7,382 @@ navigation:
 
 # Deployment - CI/CD
 
+Running tests and deployments on your local machine works great during development, but let's be honest - what we really want is for these processes to happen automatically when we push our code. This guide explains how Continuous Integration (CI) and Continuous Deployment (CD) work in a multistore setup, helping you create an efficient and reliable deployment process that runs without manual intervention.
 
-::card{title="Next: CLI Reference" icon="tabler:number-3-small" }
+**What You'll Learn**
+
+::list{type="success"}
+- How to set up CI/CD pipeline using GitHub Actions
+- How to detect and process only changed stores
+- How to efficiently deploy multiple stores in parallel
+- Advanced topics like change detection mechanism and CLI-driven approach
+::
+
+## Core Concepts
+
+### What is CI/CD in Multistore?
+
+In a multistore setup, CI/CD is designed to:
+1. Automatically detect which stores have changed
+2. Run tests only for affected stores
+3. Build and deploy only the changed stores
+
+This targeted approach ensures:
+- Faster deployments (only changed stores are processed)
+- Efficient resource usage
+- Reduced redundancy (unchanged stores are skipped in CI/CD pipeline)
+
+## Continuous Integration (CI)
```

Comment:
Having all the steps listed here and explained briefly would be nice. An image visualizing this would be superb ;)